changelog/61275.fixed

```diff
@@ -0,0 +1 @@
+Removed transport kwarg from salt.utils.event.get_event
```

Comment:
**suggestion** this should be `61275.removed`

Reply:
You know, I didn't know that was a thing.  I'll change it now.